src/Pages/VolunteerPage.jsx

```diff
@@ -195,7 +195,7 @@ const VolunteerPage = () => {
                 >
                   If you want further information and want to have a chat, don’t
                   hesitate to contact us by email at{" "}
-                  <a href="/contact">contact@cvp.com</a> and we’ll get back to
+                  <a href="/contact">convictionvacation@gmail.com</a> and we’ll get back to
```

Comment:
Could we make this the yellow color in our theme? I think it'll help make it pop and the blue is a little hard to see on the blue background

Reply:
yes! I have been thinking we need to expand that yellow into some of the other areas.